WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Carriers and Rates/ShippingLabelCarriersAndRatesViewModel.swift

```diff
@@ -0,0 +1,82 @@
+import UIKit
+import SwiftUI
+import Yosemite
+
+final class ShippingLabelCarriersAndRatesViewModel: ObservableObject {
```

Comment:
⚠️ Are we going to add a unit test for this later?

Reply:
Alrighty! In subsequent PRs is fine. I just wasn't sure if it was intentional or not. Sorry about that! 🙂